ext/native/thin3d/thin3d_d3d11.cpp

```diff
@@ -237,9 +243,24 @@ D3D11DrawContext::D3D11DrawContext(ID3D11Device *device, ID3D11DeviceContext *de
 		dxgiDevice->Release();
 	}
 	CreatePresets();
+
+	D3D11_TEXTURE2D_DESC packDesc{};
+	packDesc.CPUAccessFlags = D3D11_CPU_ACCESS_READ;
+	packDesc.BindFlags = 0;
+	packDesc.Width = 512;  // 512x512 is the maximum size of a framebuffer on the PSP.
```

Comment:
Still noting that this may not be the largest framebuffer (considering stride makes it "wider" especially), although it's the largest texture.  I think it's handled below now anyway... so just the comment is outdated.

-[Unknown]

Reply:
Right, replacing the comment.